.changeset/all-stars-travel.md

```diff
@@ -0,0 +1,38 @@
+---
+"@biomejs/biome": patch
+---
+Added the nursery lint rule `useIncludes`.
+
+This rule enforces the use of `.includes()` over `.indexOf()` when checking for the presence of an element. It applies to built-in types like `Array` and `String`, as well as any user-defined object that has both `indexOf` and `includes` methods.
+
+It also flags the use of simple regular expressions with `.test()` in favor of `String.prototype.includes()`.
+
+Using `.includes()` is more readable and modern, as it was introduced in ES2015 (for Strings) and ES2016 (for Arrays) to simplify existence checks.
```

Comment:
Let's keep the changeset also short + a reference to the rule docs

Example; https://github.com/biomejs/biome/blob/13c8082d221062b345a5d3c331e584d924a5a755/.changeset/all-kiwis-poke.md